docs/release-notes/8.0.42-33.md

```diff
@@ -0,0 +1,115 @@
+
+* [`--rocksdb_compaction_readahead_size`](https://docs.percona.com/percona-server/8.0/variables.html#rocksdb_compaction_readahead_size) - the default value is changed from `0` (zero) to `2097152`.
+
+**Deprecates MyRocks variable**
+
+* [`--rocksdb_disable_instant_ddl`](https://docs.percona.com/percona-server/8.0/variables.html#rocksdb_disable_instant_ddl) - this variable is being deprecated and is expected to be removed in a future release.
+
+**Removes MyRocks variables**
+
+* [`--rocksdb-access-hint-on-compaction-start`](https://docs.percona.com/percona-server/8.0/variables.html#rocksdb_access_hint_on_compaction_start)
+* [`--rocksdb_large_prefix`](https://docs.percona.com/percona-server/8.0/variables.html#rocksdb_large_prefix)
+* [`--rocksdb_strict_collation_check`](https://docs.percona.com/percona-server/8.0/variables.html#rocksdb_strict_collation_check)
+* [`--rocksdb_strict_collation_exceptions`](https://docs.percona.com/percona-server/8.0/variables.html#rocksdb_strict_collation_exceptions)
+
+### MySQL 8.0.42
+
+Improvements and bug fixes provided by Oracle for MySQL 8.0.42 and included in Percona Server for MySQL are the following:
+
+* Fixed an issue where `CHECK TABLE` sometimes incorrectly reported that spatial indexes were corrupted. (Bug #37286473)
+
+* Fixed an issue in InnoDB redo log recovery to improve data safety after a crash. (Bug #37061960)
+
+* Fixed an issue where reading `index_id` values could lead to incorrect behavior with indexes. (Bug #36993445, Bug #37709706)
+
+* Fixed a bug related to the `lower_case_table_names` setting that caused inconsistent behavior with table names on different systems. (Bug #32288105)
+
+* Fixed a bug where `mysqldump` did not properly escape certain special characters in its output. (Bug #37540722, Bug #37709163)
+
+* The `fprintf_string()` function in `mysqldump` did not use the correct quote character for escaping strings. (Bug #37607195)
+
+Find the complete list of bug fixes and changes in the [MySQL 8.0.42 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/8.0/en/news-8-0-42.html).
+
+## Improvements
+
+[PXC-4645](https://perconadev.atlassian.net/browse/PXC-4645): Improves State Snapshot Transfer (SST) failure diagnostics. `garbd` now uses distinct exit codes to differentiate between donor exit, SST script failure, and garbd-initiated termination, making SST issues easier to identify and debug.
+
+## Bug fixes
+
+* [PXC-3576](https://perconadev.atlassian.net/browse/PXC-3576): Percona XtraDB Cluster created role mappings using an upper-case database name when `lower_case_table_names=1` was set.
+
+* [PXC-4394](https://perconadev.atlassian.net/browse/PXC-4394): If compression was enabled for backups and State Snapshot Transfer (SST), the error log showed a misleading version comparison error. This occurred because Percona Xtrabackup was unable to return its version.
```

Comment:
Let's change the explanation to sth like:
Improved error reporting for cases when PXB used during State Snapshot Transfer (SST) was not able to report its version. It includes missing libraries or broken installation.